datafusion/sql/src/planner.rs

```diff
@@ -592,6 +602,9 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
         let schema = self.build_schema(columns)?;
 
+        // External tables do not support schemas at the moment, so the name is just a table name
```

Comment:
the parser returns a String not an ObjectIdentifier, so I kept the behavior the same